### PR TITLE
chore: update Go example

### PR DESCRIPTION
**Pull request by mdelapenya**
## What does this PR do?
It bumps testcontainers-go to the latest version, updating the test example to a consistent test case:

- a postgres database is started using the testcontainers-go's Postgres module.
- pretty print messages will appear in the case the tests pass or fail
  - PASS: testcontainers logo
  - FAIL. oh-no message

### CODE REVIEW

File: docs/active-connection.png
Comment (mdelapenya) [inline docs/active-connection.png:1]: @shelajev you probably want to remove these files from the java example too

File: README.md [hunk 1]
```diff
@@ -1,82 +1,56 @@
...
+## (optional) Use Testcontainers Desktop to easily debug the database
 
-![agent-running](./docs/active-connection.png)
+[Testcontainers Desktop](https://testcontainers.com/desktop/) helps developers with common tasks such as debugging your Testcontainers-powered dependencies. Let's practice!
 
-[tcc]: https://testcontainers.cloud/
+The tests in this project create a PostgreSQL database and populate it with sample data. You can [set a fixed port](https://newsletter.testcontainers.com/announcements/set-fixed-ports-to-easily-debug-development-services) for the `postgres` service, then [freeze containers shutdown](https://newsletter.testcontainers.com/announcements/freeze-containers-to-prevent-their-shutdown-while-you-debug) to easily connect to the database from your IDE after your tests run.
 
-[tcc-download]: https://app.testcontainers.cloud/start/download?mode=update
+See if you can inspect the database. Username: `test`. Password: `test`.
```
Comment (eddumelendez) [inline README.md:55]: See https://github.com/AtomicJar/testcontainers-cloud-go-example/pull/4/files#diff-79ce3229c13921b79b1175dcb211336aaf84dfd8edcf19c07698934d4fe70e5eR25-R26

```suggestion
See if you can inspect the database. Username: `postgres`. Password: `postgres`.
```

Comment (mdelapenya) [reply README.md]: Good catch! I used the Java file but did not check all texts

File: README.md [hunk 2]
```diff
@@ -1,82 +1,56 @@
...
-To download the agent for local usage, check the [download page here][tcc-download].
-
 ## Run the test suite
 
-`go test -v -count=1`
+Note that it's important to add the `-v` flag to the `go test` command, otherwise the output of the tests will be suppressed.
 
-### Your environment is correctly configured if
+```shell
+go test -v
```
Comment (shelajev) [inline README.md:19]: we had -count=1 before is it important?

**Review by shelajev — APPROVED** [review]

File: docs/active-connection.png
Comment (shelajev) [reply docs/active-connection.png:1]: Good point thank you!

File: README.md (hunk 2 above)
Comment (mdelapenya) [reply README.md:19]: It depends :) It disables the cache for not re-running passing tests. We can keep it for avoiding undesired errors

**Pull request merged**